test/cpp/api/parallel.cpp

```diff
@@ -212,7 +212,7 @@ TEST_F(ParallelTest, DataParallelUsesAllAvailableCUDADevices_CUDA) {
   struct M : torch::nn::Cloneable<M> {
     void reset() override {}
     torch::Tensor forward(torch::Tensor input) {
-      return torch::tensor(input.device().index());
+      return torch::tensor({input.device().index()});
```

Comment:
I changed these tensors from 0-D to 1-D, so that the original behavior of the tests are preserved.